tasks/config/webpack.config.js

```diff
@@ -16,13 +16,9 @@ module.exports = {
   },
 
   externals: {
-    classnames: true,
```

Comment:
These need to be added back in as they directly effect the `npm run build` command. Without them React is getting packaged when it shouldn't be